week1/homework/src/server.js

```diff
@@ -7,9 +7,42 @@ const http = require('http');
  */
 function createServer(port) {
   let state = 10;
+  let body;
+  let statusCode = 200;
+  let message = 'OK';
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        body = JSON.stringify({state});
+        console.log('route /state', state);
```

Comment:
You should not console.log the result:
https://github.com/HackYourFuture/Node.js/blob/master/week1/homework/README.md#endpoints-to-implement
states you should return the state in JSON format:
```
{
  "state": 10
}
```
This should not be written to the console, but to the response.